### PR TITLE
UI improved for /todo page

### LINKED ISSUE #3: Improve UI of /todo route
Improve UI of /todo route according to UI of the home page to match the theme.

THIS ISSUE DEPENDS UPON ISSUE #1. Please resolve that first.

### PR DESCRIPTION
**Pull request by amitmishra11**
@sanskaromar have a look!

Resolves #3

### CODE REVIEW

**Comment by sanskaromar** [comment]
@amitmishra11 The theme after your changes does not properly match the functionality. And that I am looking for a less fancy minimalistic view with a good color palette in mind. Please make changes accordingly.

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
+    "react-toastify": "^9.0.8",
```
Comment (sanskaromar) [inline package.json:15]: React-toastify is installed but not used anywhere.

**Review by sanskaromar — CHANGES_REQUESTED** [review]
The theme after your changes does not properly match the functionality. And I am looking for a less fancy, minimalistic view with a good color palette in mind. Please make changes accordingly.

File: package.json (hunk 1 above)
Comment (amitmishra11) [reply package.json:15]: @sanskaromar  yeah I tried using it but was not able to use it as it was throwing some error

**Comment by amitmishra11** [comment]
> The theme after your changes does not properly match the functionality. And that I am looking for a less fancy minimalistic view with a good color palette in mind. Please make changes accordingly.

What about complete gray theme as used around task boxes?

**Comment by sanskaromar** [comment]
> > The theme after your changes does not properly match the functionality. And that I am looking for a less fancy minimalistic view with a good color palette in mind. Please make changes accordingly.
> 
> What about complete gray theme as used around task boxes?

I do not think it looks good there. What you can instead do, is make UI for light theme for app. That way you won't have to worry about colors used in this theme.

**Comment by amitmishra11** [comment]
> > > The theme after your changes does not properly match the functionality. And that I am looking for a less fancy minimalistic view with a good color palette in mind. Please make changes accordingly.
> > 
> > 
> > What about complete gray theme as used around task boxes?
> 
> I do not think it looks good there. What you can instead do, is make UI for light theme for app. That way you won't have to worry about colors used in this theme.

ohk I'll do this

**Comment by amitmishra11** [comment]
@sanskaromar is this okay?

**Comment by amitmishra11** [comment]
> Please make these minor changes.

@sanskaromar surely

**Comment by amitmishra11** [comment]
@sanskaromar  Recommended changes made

**Review by sanskaromar — APPROVED** [review]
I am approving the changes. Great work 🚀

File: src/components/todoapp.js [hunk 2]
```diff
@@ -22,7 +21,7 @@ export default function Todos() {
                             }} />
                     }
 
-                    <input className="w-full rounded border-4 border-blue-300 px-2 py-1 text-blue-50 bg-gray-700 focus:bg-gray-900 focus:text-white focus:border-blue-200"
+                    <input className="w-full rounded-2xl border-4 border-blue-300 px-2 py-1 text-blue-50 bg-gray-300 focus:bg-gray-400 focus:text-white focus:border-blue-200 text-3xl"
```
[the remark this answers is not in the document]
Comment (amitmishra11) [reply src/components/todoapp.js:24]: Wait adding

**Comment by amitmishra11** [comment]
@sanskaromar It was nice working with you!

**Pull request merged**